src/Twee/School/TeacherPunishmentsScenes.twee

```diff
@@ -977,7 +991,7 @@
 		After a short drive she drags your into the store and picks a chastity device, which is smaller than your original one.
 		
 		<<if playerCode.isMind_0()>>\
-			@@.hero;Fuck, she made my penis shrink! And it's going to be permanent! And it is my own fault.@@ A feeling of utter defeat fills you. There's nothing you can do about it.
+			@@.hero;Fuck, she made my penis shrink! And it's going to be permanent! And it is my own fault, I totally brought this on myself, and have noone else to blame...@@ A feeling of utter defeat fills you. There's nothing you can do about it... even if you wanted to.
```

Comment:
```suggestion
			@@.hero;Fuck, she made my penis shrink! And it's going to be permanent! And since I told her she could, I have no one else to blame...@@ A feeling of utter defeat fills you. There's nothing you can do about it... even if you wanted to.
```